examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,80 @@
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class UnknownIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let identifier: Swift.String
+  @objc(initWithIdentifier:) public init(_ identifier: Swift.String)
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class PhoneNumberIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let phoneNumber: Swift.String
+  @objc final public let rawId: Swift.String?
+  @objc public init(phoneNumber: Swift.String, rawId: Swift.String? = nil)
+  public static func == (lhs: AzureCommunication.PhoneNumberIdentifier, rhs: AzureCommunication.PhoneNumberIdentifier) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class MicrosoftTeamsUserIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let userId: Swift.String
+  @objc final public let isAnonymous: Swift.Bool
+  @objc final public let rawId: Swift.String?
+  @objc final public let cloudEnviroment: AzureCommunication.CommunicationCloudEnvironment
+  @objc public init(userId: Swift.String, isAnonymous: Swift.Bool = false, rawId: Swift.String? = nil, cloudEnvironment: AzureCommunication.CommunicationCloudEnvironment = .Public)
+  @objc public init(userId: Swift.String, isAnonymous: Swift.Bool)
+  public static func == (lhs: AzureCommunication.MicrosoftTeamsUserIdentifier, rhs: AzureCommunication.MicrosoftTeamsUserIdentifier) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class CommunicationAccessToken : ObjectiveC.NSObject {
+  @objc final public let token: Swift.String
+  @objc final public let expiresOn: Foundation.Date
+  @objc public init(token: Swift.String, expiresOn: Foundation.Date)
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class CommunicationCloudEnvironment : ObjectiveC.NSObject {
+  @objc public static var Public: AzureCommunication.CommunicationCloudEnvironment
+  @objc public static var Dod: AzureCommunication.CommunicationCloudEnvironment
+  @objc public static var Gcch: AzureCommunication.CommunicationCloudEnvironment
+  @objc public init(environmentValue: Swift.String)
+  @objc public func getEnvironmentValue() -> Swift.String
+  public static func == (lhs: AzureCommunication.CommunicationCloudEnvironment, rhs: AzureCommunication.CommunicationCloudEnvironment) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+public typealias TokenRefresherClosure = (@escaping AzureCommunication.TokenRefreshOnCompletion) -> Swift.Void
+@objc @objcMembers public class CommunicationTokenRefreshOptions : ObjectiveC.NSObject {
+  @objc public init(initialToken: Swift.String? = nil, refreshProactively: Swift.Bool = false, tokenRefresher: @escaping AzureCommunication.TokenRefresherClosure)
```

Comment:
Is the `TokenRefreshClosure` something that does the magic of refreshing the token, or is it called when a token is refreshed?

Reply:
This is called when the token is refreshed

Reply:
This is contosos method that we will call to return a new token.